source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -162,37 +162,17 @@ Install Concordium Ledger app on macOS
 Install Homebrew, Python3, and pip
 ----------------------------------
 
-#. Open the Terminal application. Click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
+#. To open the Terminal application, click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
```

Comment:
The user might not have the Launchpad in their dock. It's just a shortcut, and they could have removed it.
And users might not know that the grid of apps is called the launchpad. (I know that I wrote this, but we might as well update it if we can come up with a better phrase 😊  )

Reply:
I'm not a Mac user. So what would a user need to do if Launchpad isn't in their dock? Or does Apple have a support article we could link to? That would be the best.

Reply:
Great idea with the link to Apple's own terminal guide ;)